lib/recurly/coupon.php

```diff
@@ -56,6 +68,17 @@ public static function deleteCoupon($couponCode, $client = null) {
     return Recurly_Base::_delete(Recurly_Coupon::uriForCoupon($couponCode), $client);
   }
 
+  protected function createUpdateXML() {
```

Comment:
I think we had discussed in chat that passing the HTTP method down that low to the serialization was too messy.